gulpfile.js

```diff
@@ -234,9 +254,9 @@ gulp.task('clean-build-app-prod', ['clean-prod'], pipes.builtAppProd);
 gulp.task('prod-artifacts', ['clean-build-app-prod'], pipes.buildArtifacts);
 
 // clean, build, and watch live changes to the dev environment
-gulp.task('watch-dev', ['build-styles-dev', 'validate-app-scripts'], function () {
+gulp.task('watch-dev', ['build-styles-dev', 'validate-app-scripts', 'bump'], function () {
```

Comment:
If we're bumping the version number every time we run gulp, we want to add this bump to watch-prod and prod-artifacts -- prod-artifacts being the most important because that's what is being called on deployment.